services/accumulo-service/src/main/java/datawave/microservice/audit/AuditClient.java

```diff
@@ -0,0 +1,196 @@
+package datawave.microservice.audit;
```

Comment:
This seems like another candidate for a shared library of some kind. When we get to any query type of service, we'll need to send audits from them too.